mindmeld/components/nlp.py

```diff
@@ -1286,22 +1295,30 @@ def _recognize_entities(self, query, dynamic_resource=None, verbose=False):
                 return nbest_transcripts_entities
             else:
                 if verbose:
+                    if len(self.entities) == 0:
```

Comment:
I think you can add one statement on top of the conditional. I dont think you need to worry about `[[]]` or `[()]`. lets stick with the latter.